src/main/java/net/dv8tion/jda/api/managers/ChannelManager.java

```diff
@@ -235,10 +235,114 @@ default ChannelManager putPermissionOverride(@Nonnull IPermissionHolder permHold
         return putPermissionOverride(permHolder, allowRaw, denyRaw);
     }
 
+    /**
+     * Adds an override for the specified role with the provided raw bitmasks as allowed and denied permissions. If the
+     * role already had an override on this channel it will be replaced instead.
```

Comment:
```suggestion
     * Adds an override for the specified role with the provided raw bitmasks as allowed and denied permissions.
     * If the role already had an override on this channel it will be replaced instead.
```